src/state/store.ts

```diff
@@ -103,8 +105,8 @@ const stateSanitizer = (state: any) => {
 }
 
 /// This allows us to create an empty store for tests
-export const createStore = () =>
-  configureStore({
+export const createStore = () => {
+  return configureStore({
```

Comment:
why not but also why

Reply:
https://github.com/shapeshift/web/pull/4209/commits/964445330d4e93772f56ab276f660af181708865